code/headers/uart_ports.hpp

```diff
@@ -2,49 +2,13 @@
 #include <hwlib.hpp>
 
 namespace r2d2 {
-    
-    
-    enum class uart_ports_c {
-        uart0, uart1, uart2, uart3, 
+
+    enum class uart_ports_c : uint8_t {
+        uart1 = 0,
+        uart2 = 1,
+        uart3 = 2,
         // only for determining how many uart ports there are
         UART_SIZE
 
     };
-
```

Comment:
With this wip/commented code gone, shouldn't  this enum class be moved to usart_connection.hpp?

Reply:
also uart_ports is incorect, should be usart_ports my bad

Reply:
sure I'm gonna move it in PR #13